diagnostic.md

```diff
@@ -27,7 +27,9 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  font-size: ($base-font-size * .30);
```

Comment:
This will decrease the font size to 30% of `$base-font-size`'s value.  Since we want to increase the size by 30%, it should be `font-size: $base-font-size * 1.3;`.